utils/frame/try-runtime/cli/src/lib.rs

```diff
@@ -265,6 +265,8 @@
 //!     -s snap \
 //! ```
 
+#![cfg(feature = "try-runtime")]
```

Comment:
This should be fine, or? @kianenigma

Reply:
makes sense to me! If `try-runtime` isn't built, the CLI should just print an error to compile with the flag